OpenRA.Game/Game.cs

```diff
@@ -250,16 +250,25 @@ internal static void Initialize(Arguments args)
 			Log.AddChannel("geoip", "geoip.log");
 			Log.AddChannel("irc", "irc.log");
 
-			if (Settings.Server.DiscoverNatDevices)
-				UPnP.TryNatDiscovery();
-			else
+			GeoIP.Initialize();
```

Comment:
Please keep all this *after* the renderer initialization and first load screen paint.  Doing it here increases the delay between running the game and having the window and load screen first appear.